Update machine-learning-udf.md (#117075)
Azure/azure-content: articles/stream-analytics/machine-learning-udf.md

* Not Found (404)
* Unauthorized (401)

## Limitations

> If you are using an Azure ML Managed Endpoint service, Stream Analytics can currently only access enpoints that have public network access enabled. Read more about it on the page about [Azure ML private endpoints](https://learn.microsoft.com/en-us/azure/machine-learning/concept-secure-online-endpoint?view=azureml-api-2&tabs=cli#secure-inbound-scoring-requests)

Reviewer: ```suggestion
If you're using an Azure ML Managed Endpoint service, Stream Analytics can currently only access endpoints that have public network access enabled. Read more about it on the page about [Azure ML private endpoints](/azure/machine-learning/concept-secure-online-endpoint?view=azureml-api-2&tabs=cli#secure-inbound-scoring-requests).
```
